lib/pacemaker/pcmk_output.c

```diff
@@ -1999,6 +2002,21 @@ cluster_status_xml(pcmk__output_t *out, va_list args)
                      false);
     }
 
+
+    /* If there are verification errors, always print a statement about that, even if not requested */
+
+    pcmk__output_new(&verify_out, "none", NULL, NULL);
+    verify_rc = pcmk__verify(scheduler, verify_out, scheduler->input);
+    pcmk__output_free(verify_out);
+
+    if (verify_rc == pcmk_rc_ok) {
+        if (pcmk_is_set(section_opts, pcmk_section_verify)) {
+            out->info(out, "CIB syntax is valid");
+        }
+    } else {
+        out->info(out, "CIB syntax has errors (for details, run crm_verify -LV).");
```

Comment:
info() does nothing for XML, you have to add XML elements or attributes with the info